server/worker/send_gmail.py

```diff
@@ -15,6 +15,8 @@
 DB_ADDR = os.environ.get("DB_ADDR", None)
 DB_ALIAS = "user-db"
 
+BATCH_SIZE = 1
```

Comment:
This is where we change how many emails before an event is sent, correct?

Reply:
Yes